doc/source/_apidoc/qutip_qip.decompositions.rst

```diff
@@ -0,0 +1,37 @@
+qutip\_qip.decompositions package
```

Comment:
HTML build failing due to renamed modules. Change before requesting a review.

Reply:
Yeah, it should be fairly easy to update this using the command https://qutip-qip.readthedocs.io/en/latest/contribution-docs.html#generate-api-documentation.